src/main/target/STM32F405/config/ALIENFLIGHTNGF4.config

```diff
@@ -49,6 +49,7 @@ resource SDCARD_DETECT 1 B11
 set sdcard_detect_inverted = ON
 set sdcard_mode = SPI
 set sdcard_spi_bus = 2
+dmaopt SPI_TX 2 0
```

Comment:
Looking at this, I think we need to address the topic of capitalisation in target configurations - defining and enforcing a unified style will make it easier to read and grep / filter target configurations, and eliminate churn by capitalisation changes.

I would tend to go for making 'all non-comment lines of target configurations shall be lower case'. This is also in line with the way that CLI `diff` prints the lines.